### PR TITLE
Feature/10203 fss UI  search query editor  add intelligence

### PR DESCRIPTION
**Pull request by Shrikan14443**
code changes to add intelligence for operator field and value input field.

### CODE REVIEW

File: src/app/core/services/fss-search-filter.service.ts [hunk 1]
```diff
@@ -15,35 +22,48 @@ export class FssSearchFilterService {
     for(var i=0; i < fssSearchRows.length; i++) {
 
       var fssSearchRow = fssSearchRows[i];
+      // getFieldDataType
+      const dataType = this.getFieldDataType(fssSearchRow);
```
Comment (bhalcha13813) [inline src/app/core/services/fss-search-filter.service.ts:26]:
```suggestion
      const dataType = this.getFieldDataType(fssSearchRow);
```
```suggestion
      const fieldDataType = this.getFieldDataType(fssSearchRow);
```

Comment (Shrikan14443) [reply src/app/core/services/fss-search-filter.service.ts]: Suggested changes has been done.

**Review by bhalcha13813 — APPROVED** [review]
LGTM

**Review by perrytom — APPROVED** [review]
Looks good to me.

**Pull request merged**